common/src/main/java/com/microsoft/identity/common/internal/cache/MicrosoftFamilyOAuth2TokenCache.java

```diff
@@ -135,7 +139,7 @@ public ICacheRecord loadByFamilyId(@Nullable final String clientId,
             }
         }
 
-        if (null != target) {
+        if (null != target && null != authenticationScheme) {
```

Comment:
This value should never be null in AT case i think (as it is passed from the outer layer). The one in cache, on the other hand, can be null.

Reply:
Yup, this check is for the passed-in value. It won't be null unless explicitly provided as null (so that it behaves as a wildcard (*) which _is_ supported from an API perspective)